### DISCUSSION TITLE
Is it possible to make clutter_offscreen_effect_get_target() available to CJS?

### DISCUSSION THREAD

**Discussion by klangman** [linuxmint/discussions#610] [Development]
Wondering what it would take to get this function available to CJS for use by an extension?

https://github.com/linuxmint/muffin/blob/9aefdd415cf3ac152bfd9563b3b799ac5e1393f7/clutter/clutter/clutter-offscreen-effect.c#L598

**Comment by Fantu** [comment]
from a fast look to [muffin symbols](https://github.com/linuxmint/muffin/blob/master/debian/libmuffin0.symbols) clutter_offscreen_effect_get_target is already exposed

**Comment by klangman** [comment]
@Fantu 
```
(cinnamon:2697): Gjs-CRITICAL **: 09:36:21.060: JS ERROR: TypeError: shader.get_target is not a function
Effect/this.shaderFactory</<@/home/kevin/.local/share/cinnamon/extensions/CinnamonBurnMyWindows@klangman/6.2/effects/BrokenGlass.js:122:33
vfunc_paint_target@/home/kevin/.local/share/cinnamon/extensions/CinnamonBurnMyWindows@klangman/6.2/Shader.js:249:12
vfunc_paint@/usr/share/cinnamon/js/ui/layout.js:96:19
```

class Shader extends Cinnamon.GLSLEffect

Doesn't seem to work called from javascript.

**Comment by klangman** [comment]
I think it could be that the GObject-Introspection annotations for clutter_offscreen_effect_get_target has "(skip)" specified?
